pkg/minikube/download/binary.go

```diff
@@ -53,7 +53,11 @@ func binaryWithChecksumURL(binaryName, version, osName, archName, binaryURL stri
 
 // Binary will download a binary onto the host
 func Binary(binary, version, osName, archName, binaryURL string) (string, error) {
-	targetDir := localpath.MakeMiniPath("cache", osName, archName, version)
+	// targetDir := localpath.MakeMiniPath("cache", osName, archName, version)
+	targetDir := localpath.MakeMiniPath("cache", "binaries", osName, archName, version)
```

Comment:
I think we should use "bin", it is more consistent with ~/.minikube/bin and the rest of the world.

We are using "binaries" internally which would nice to change also to bin but it may be harder since we may code using it. Since this cache directory is new we don't have backward compatibility concerns.